src/livecodes/UI/assets.ts

```diff
@@ -135,6 +144,7 @@ const organizeAssets = async (
     '#assets-list-container #assets-reset-filters',
   ) as HTMLElement;
 
+  // TODO: i18n this
```

Comment:
Any comments on asset type i18n?

Reply:
I'm not sure.

The types are [listed here](https://github.com/zyf722/livecodes/blob/8345e09aa7ea2cdc284537293bff0243d0efa190/src/livecodes/UI/assets.ts#L690).
However, when a new asset is added, its type, along with other asset data are stored in the browser storage.
We have agreed before that we will not translate user data, like project title, starter template content, etc.
This carries the possibility that the data is saved in one language, then the language is changed and the data now gets displayed in a wrong language.

Having said that, another option would be that we still save it in English, but when we display it we use the saved asset type as the translation key, and display it with the UI language.

I'm fine with both approaches.

Reply:
We also need to translate the word "Date:" [here](https://github.com/zyf722/livecodes/blob/8345e09aa7ea2cdc284537293bff0243d0efa190/src/livecodes/UI/assets.ts#L70)

Reply:
Fixed in https://github.com/zyf722/livecodes/commit/abe4671.

> Another option would be that we still save it in English, but when we display it we use the saved asset type as the translation key, and display it with the UI language.

A [`typesToShow`](https://github.com/zyf722/livecodes/blob/i18next-integration-draft/src/livecodes/UI/assets.ts#L33-L48) mapping is added to do this. Have to leave some explicit code duplication here to make `i18n-export` work :(

Reply:
Not pretty, but works!